types/underscore/index.d.ts

```diff
@@ -6498,6 +6582,52 @@ declare namespace _ {
          */
         isUndefined(): _ChainSingle<boolean>;
 
+        /**
+         * Returns true if the wrapped object is a Set.
+         * @returns True if the wrapped object is a Set, otherwise false.
+         * The result will be wrapped in a chain wrapper.
+         */
+        isSet(): _ChainSingle<boolean>;
```

Comment:
Why not `_ChainSingle<V is Set<any>>`?

Reply:
Same. Tsc gives error on that, I think type predicate in this way cannot be done.